source/_integrations/zwave_js.markdown

```
@@ -1093,21 +1094,21 @@ Set the log level for `zwave_js_server` to a level higher than `debug`. This can
 
-### Including / excluding a controller in an existing network using [classic inclusion](#classic-inclusion-versus-smartstart)
+### Including / excluding a adapter in an existing network using [classic inclusion](#classic-inclusion-versus-smartstart)
 
 A Z-Wave controller that manages an empty network can also join a different network and act as a secondary controller there. However, with Home Assistant, this is not possible. Home Assistant does not allow the Z-Wave controller to join another network, because Home Assistant acts as the central hub.
 
 ## Z-Wave association groups
 
 In Home Assistant, a single [association group](#association-group) is implemented:
 
-- **Group 1**: This is an association group that includes only one device. It is used after a [factory reset](#controller), to send a **Device Reset Locally Notification**.
+- **Group 1**: This is an association group that includes only one device. It is used after a [factory reset](#adapter), to send a **Device Reset Locally Notification**.
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)